src/xrpld/app/tx/detail/MPTokenIssuanceSet.cpp

```diff
@@ -152,6 +220,42 @@ MPTokenIssuanceSet::preclaim(PreclaimContext const& ctx)
         }
     }
 
+    auto const currentMutableFlags =
+        (*sleMptIssuance)[~sfMutableFlags].value_or(0);
```

Comment:
since `sfMutableFlags` field of `MPTokenIssuance` is no longer `soeOPTIONAL` but `soeDEFAULT`, this code is now unnecessarily complex and should be simplified.